DigitalLearningSolutions.Web/Services/RegistrationService.cs

```diff
@@ -349,7 +378,19 @@ public void CreateCentreManagerForExistingUser(int userId, int centreId, string?
                 userAccount.ProfileImage
             );
 
-            registrationDataService.RegisterAdmin(new AdminAccountRegistrationModel(registrationModel, userId));
+            registrationDataService.RegisterAdmin(
+                new AdminAccountRegistrationModel(registrationModel, userId),
+                new PossibleEmailUpdate
+                {
+                    OldEmail = userDataService.GetCentreEmail(userId, centreId),
+                    NewEmail = centreSpecificEmail,
+                    NewEmailIsVerified = emailVerificationDataService.AccountEmailIsVerifiedForUser(
+                                             userId,
+                                             centreSpecificEmail
+                                         ),
+                    CentreId = null,

```

Comment:
I don't think that `CentreId` should be null here, but also given the other changes I am suggesting, we can remove the need for `PossibleEmailUpdate.CentreId`